### PR TITLE
Updating all samples to Compose 1.1.0-rc01 and Kotlin 1.6.10

### PR DESCRIPTION
**Pull request by florina-muntenescu**

### CODE REVIEW

**Review by Umer19 — CHANGES_REQUESTED** [review]
dev-rc02

**Comment by Umer19** [comment]
#716

**Review by Umer19 — CHANGES_REQUESTED** [review]
#714

File: Jetchat/app/src/main/java/com/example/compose/jetchat/components/JetchatScaffold.kt [hunk 1]
```diff
@@ -16,25 +16,25 @@
...
-import androidx.compose.material3.ScaffoldState
-import androidx.compose.material3.rememberScaffoldState
+import androidx.compose.material3.NavigationDrawer
+import androidx.compose.material3.rememberDrawerState
 import androidx.compose.runtime.Composable
 import com.example.compose.jetchat.theme.JetchatTheme
 
 @OptIn(ExperimentalMaterial3Api::class)
 @Composable
 fun JetchatScaffold(
```
Comment (nickbutcher) [inline Jetchat/app/src/main/java/com/example/compose/jetchat/components/JetchatScaffold.kt:29]: We should probably rename this as it's not a `Scaffold` any more?

**Review by nickbutcher — APPROVED** [review]

File: Jetchat/app/src/main/java/com/example/compose/jetchat/components/JetchatScaffold.kt (hunk 1 above)
Comment (JoseAlcerreca) [reply Jetchat/app/src/main/java/com/example/compose/jetchat/components/JetchatScaffold.kt:29]: https://github.com/android/compose-samples/issues/720

**Pull request merged**